### PR TITLE
aliasing any jcr-namespaced properties to fedora:

### PR DESCRIPTION
**Pull request by acoburn**
https://www.pivotaltracker.com/n/projects/684825/stories/83411334

any `jcr:` properties are transformed to `fedora:` before sending to the JMS broker.

This potentially causes **any** `jcr:` property to populate the `fedora:` namespace, which may not be desired.

### CODE REVIEW

**Comment by acoburn** [comment]
This is now tracked in JIRA: https://jira.duraspace.org/browse/FCREPO-1405

File: fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/observer/GetNamespacedProperties.java [hunk 1]
```diff
@@ -54,11 +57,21 @@ public FedoraEvent apply(final FedoraEvent evt) {
...
                 final String prefix = parts[0];
-                try {
-                    event.addProperty(namespaceRegistry.getURI(prefix) + parts[1]);
-                } catch (RepositoryException ex) {
-                    LOGGER.trace("Prefix could not be dereferenced using the namespace registry: {}", property);
-                    event.addProperty(property);
+                if ("jcr".equals(prefix)) {
+                    if (jcrProperties.contains(createProperty(REPOSITORY_NAMESPACE + parts[1]))) {
+                        event.addProperty(REPOSITORY_NAMESPACE + parts[1]);
+                    } else if (LOGGER.isDebugEnabled()) {
```
Comment (awoods) [inline fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/observer/GetNamespacedProperties.java]: Please remove the `LOGGER.isDebugEnabled()` guard to make it easier to change the logging state at runtime.

File: fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/observer/GetNamespacedProperties.java [hunk 2]
```diff
@@ -54,11 +57,21 @@ public FedoraEvent apply(final FedoraEvent evt) {
...
+                        LOGGER.debug("Swallowing jcr property: {}", property);
+                    }
+                } else {
+                    try {
+                        event.addProperty(namespaceRegistry.getURI(prefix) + parts[1]);
+                    } catch (RepositoryException ex) {
+                        if (LOGGER.isDebugEnabled()) {
```
Comment (awoods) [inline fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/observer/GetNamespacedProperties.java]: Please remove the `LOGGER.isDebugEnabled()` guard to make it easier to change the logging state at runtime.

Comment (ajs6f) [inline fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/observer/GetNamespacedProperties.java]: There's really just no reason to use these kinds of tests. They are built into the logging framework. Just `LOGGER.debug()` whatever it is and the logging framework will decide whether or not to execute the action.

**Comment by awoods** [comment]
Resolved with: https://github.com/fcrepo4/fcrepo4/commit/be0e046cdcbc0653a2958c044c6ccd34e8e0f6c8

**Pull request closed without merging**